v1: PostgreSql upgrade

AlexKalnitskiy opened the pull request:
Migrate EventFlow.PostgreSql package from v0 to v1

- Move ILog to ILogger<>
- Added Microsoft DI support
- Updated dependencies
- Upgraded tests

AlexKalnitskiy committed:
Upgrade PostgreSql to v1.

AlexKalnitskiy committed:
fixed loggers

AlexKalnitskiy committed:
upgraded postgre sql tests to v1

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/eventflow/EventFlow?pullRequest=918) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/eventflow/EventFlow?pullRequest=918) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/eventflow/EventFlow?pullRequest=918) it.</sub>

AlexKalnitskiy committed:
Changed di behavior to TryAdd*

AlexKalnitskiy committed:
small fix

changed behavior in EventFlowOptionsPostgreSqlReadStoreExtensions, now it uses GetRequiredService instead of GetService

rasmus committed:
Merge branch 'develop-v1' into v1-postgresql-upgrade

rasmus committed:
Update VS solution and readme

rasmus committed:
Merge branch 'develop-v1' into v1-postgresql-upgrade

rasmus commented:
Sorry for the delay, thanks for the PR @AlexKalnitskiy
